test/e2e/README.md

```diff
@@ -0,0 +1,30 @@
+# E2E Test
+
+This document is to help developers understand how to run e2e test CAPDO.
```

Comment:
```suggestion
This document is to help developers understand how to run e2e test for CAPDO.
```
Nit: I think it would read better with `for`.

Reply:
updated